src/microbe_stage/components/CellProperties.cs

```diff
@@ -532,6 +532,9 @@ public static bool IsMembraneReady(this ref CellProperties cellProperties)
             cellProperties.Colour = newDefinition.Colour;
             cellProperties.MembraneRigidity = newDefinition.MembraneRigidity;
 
+            // Sets name; uses baseReproductionCostFrom as it is species
+            entity.Get<ReadableName>().Name = new LocalizedString(baseReproductionCostFrom.FormattedName);
```

Comment:
This should have a safety check to ensure the component exists as it is not critical to the primary function of this method.